app/styles/modules/_password-row.scss

```diff
@@ -20,10 +20,20 @@
 
     html[dir=ltr] & {
       text-align: left;
+
+      &.input-help-focused {
+        text-align: center;
+      }
+
     }
 
     html[dir=rtl] & {
       text-align: right;
+
+      &.input-help-focused {
+        text-align: center;
+      }
```

Comment:
Need the `&.input-help-focused` inside `html[dir=x]` for specificity. Otherwise the centering never kicks in. Can move it to something like this:
```
html[dir=rtl] &,
html[dir=ltr] & {
&.input-help-focused {
        text-align: center;
      }
}
```

which would repeat the selectors again (adding the same 3 loc, resulting in the same output). I think we should keep it this way, so that if we want different styles for ltr and rtl, it can be done.